src/graphics/clouds.c

```diff
@@ -269,8 +271,16 @@ void clouds_draw(int x_offset, int y_offset, int x_limit, int y_limit, float bas
             cloud->status = STATUS_INACTIVE;
             continue;
         }
-        speed_set_target(&cloud->speed.x, -CLOUD_SPEED, SPEED_CHANGE_IMMEDIATE, 1);
-        speed_set_target(&cloud->speed.y, CLOUD_SPEED / 2, SPEED_CHANGE_IMMEDIATE, 1);
+
+        double cloudSpeed = 0;
+
+        if (!game_state_is_paused()) {
+            double speedShift = (100 - setting_game_speed()) / 100;
+            cloudSpeed = CLOUD_SPEED - speedShift;
```

Comment:
```suggestion
            double speed_shift = (100 - setting_game_speed()) / 100;
            cloud_speed = CLOUD_SPEED - speed_shift;
```

Aaaahhhh you're giving me an hearth attack!!! 🤣